README.md

```diff
@@ -0,0 +1,23 @@
+# Opdrachtbeschrijving
+
+In deze repository vind je een aantal opdrachten die je kunt maken om te oefenen met lists en tuples in Python.
+Deze repository bestaat uit meerdere opdrachten van oplopende moeilijkheidsgraad. 
+
+Om deze opdracht te maken kun je op Github, onder de groene knop 'Code', deze repository downloaden als zip bestand en openen in je IDE
```

Comment:
Zou je zin willen vervangen met:

Je kunt deze opdracht maken door het project te downloaden als ZIP of met Git te clonen naar jouw eigen computer. Heb jij Git niet geïnstalleerd en download je het project als ZIP-map? Pak de ZIP-map eerst uit nadat deze is gedownload naar jouw Downloads-map. Windows-gebruikers doen dit [via deze stappen](https://www.youtube.com/watch?v=s4_WdMSaKRE), Mac-gebruikers hoeven slechts op de map te dubbelklikken. Daarna kun je het project openen in PyCharm (_File > Open.._).